### PR TITLE
finos/a11y-theme-builder#903:solve and check the that lists are struc…

### PR DESCRIPTION
**Pull request by ShadabImran07**
…tured correctly (ul)

### CODE REVIEW

**Comment by ShadabImran07** [comment]
@aaronreed708 created pr for this issue check it and let me know if anything else is required

**Comment by ShadabImran07** [comment]
@aaronreed708 created pr for this issue check it and let me know if anything else is required

File: code/src/data/themes
Comment (aaronreed708) [inline code/src/data/themes:1]: nit: we don't usually check in code/src/data/themes unless we added a new, default theme for everyone to use.  It is just a DB of themes and will often show up as changed on in your local repo if you use non-default themes.  Just make sure that you create your PRs without it.  Thanks!

File: code/src/ui/debug.log
Comment (aaronreed708) [inline code/src/ui/debug.log:1]: another file that you shouldn't be committing :-)

File: code/src/ui/src/components/LeftNavTabs.tsx [hunk 1]
```diff
@@ -2,64 +2,66 @@
...
 
-export const LeftNavText = styled(ListSubheader)(( {theme }) => ({
-    // textTransform: "uppercase",
-    background: "transparent",
-    fontSize: "1em",
+export const LeftNavText = styled(ListSubheader)(({ theme }) => ({
+	// textTransform: "uppercase",
+	background: "transparent",
+	fontSize: "1em",
 }));
```
Comment (aaronreed708) [inline code/src/ui/src/components/LeftNavTabs.tsx]: Looks like your editor is changing formatting in existing files.  When contributing to existing files in open source, you should follow the formatting and styling that already exists.  And if/when contributing new files that you created, you should try to conform to the styling that the rest of the project is using.  There is already an effort underway to add Prettier, but that contribution isn't ready, yet.

**Review by aaronreed708 — CHANGES_REQUESTED** [review]
Thank you for your contribution!  It is too hard to tell what your changes are with all of the formatting and styling changes.  Please remove the extra files from your contribution and the formatting and styling changes so that it will be more clear what your action solution is.  Thank you!

**Pull request closed without merging**